_posts/2024-10-08-cockpit-326.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Cockpit 326
+author: mvo
+date: '2024-10-08'
+tags: cockpit, podman, files
+slug: cockpit-326
+category: release
+summary: 'The cockpit/ws container now supports connections to hosts without Cockpit installed and has ssh key support, cockpit-pcp packagge replaced with native PCP support in bridge, Storage gains support for the Stratis virtual filesystem, cockpit-podman supports registries without a search API, and cockpit-files gains a keyboard shortcuts dialog'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 326, cockpit-podman 96, and cockpit-files 9:
+
+## cockpit/ws container: Connect to any Linux server
+
+The [quay.io/cockpit/ws](https://quay.io/repository/cockpit/ws) container provides Cockpit's web server in environments such as Kubernetes and [Fedora CoreOS](https://fedoraproject.org/coreos/), connecting to a remote machine via `ssh`.
```

Comment:
I'm not sure how it happens, as md is correct - but the rendered view is missing a space:

![image](https://github.com/user-attachments/assets/29223660-06ab-4af7-8008-9a010bd5f735)